jekyll/_cci2/concepts.md

```diff
@@ -583,6 +583,21 @@ On the Projects Dashboard, you can either:
 
 ![header]({{ site.baseurl }}/assets/img/docs/CircleCI-2.0-setup-project-circle101_cloud.png)
 
+## Resource class
+{: #resource-class}
+
+A resource class is a configuration option that allows you to control available compute resources (CPU and RAM) for your jobs. When you specify an execution environment for a job, a default resource class value for the environment will be set _unless_ you define the resource class in your [configuration]({{site.baseurl}}/2.0/configuration-reference#resourceclass). It is best practice to define the resource class, as opposed to relying on a default.
+
```

Comment:
```suggestion

The example below shows how to define a resource class in the Docker execution environment. Examples for all execution environments are available on the following pages:

* [Using the Docker execution environment]({{site.baseurl}}/2.0/using-docker)
* [Using the LinuxVM execution environment]({{site.baseurl}}/2.0/using-linuxvm)
* [Using the macOS execution environment]({{site.baseurl}}/2.0/using-macos)
* [Using the Windows execution environment]({{site.baseurl}}/2.0/using-windows)
* [Using the GPU execution environment]({{site.baseurl}}/2.0/using-gpu)
* [Using the Arm execution environment]({{site.baseurl}}/2.0/using-arm)

Pricing and plans information for the various resource classes can be found [here](https://circleci.com/product/features/resource-classes/).

```

Reply:
Just a suggestion to ensure people can quickly get to resource class info for their environment. Might seem like too much here so I don't mind if you don't want to include. I just notice in our general docs feedback that people struggle to infer config for their environment from an example based on an alternative, it that makes sense. And our docs on execution envs are a bit confusing generally 🤷‍♀️

Reply:
Also not sure we need the 2.0 in these links anymore!